qark/plugins/webview/javascript_enabled.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+import javalang
+from javalang.tree import MethodInvocation
+
+from qark.issue import Severity, Issue
+from qark.plugins.helpers import java_files_from_files
+from qark.plugins.webview.helpers import valid_set_method_bool
+from qark.scanner.plugin import BasePlugin
+
+log = logging.getLogger(__name__)
+
+JAVASCRIPT_ENABLED_DESCRIPTION = (
+    "While not a vulnerability by itself, it appears this app has JavaScript enabled in "
+    "this WebView: localWebView If this is not expressly necessary, you should disable "
```

Comment:
Is `localWebView` meant to be substituted for a variable?

Reply:
Yeah I think it was broken from the original QARK code. Removed.